src/node_crypto.cc

```diff
@@ -3807,12 +3808,18 @@ void Hmac::HmacDigest(const FunctionCallbackInfo<Value>& args) {
     md_len = 0;
   }
 
-  Local<Value> rc = StringBytes::Encode(env->isolate(),
-                                        reinterpret_cast<const char*>(md_value),
-                                        md_len,
-                                        encoding);
+  Local<Value> error;
+  MaybeLocal<Value> rc =
+      StringBytes::Encode(env->isolate(),
+                          reinterpret_cast<const char*>(md_value),
+                          md_len,
+                          encoding,
+                          &error);
   delete[] md_value;
-  args.GetReturnValue().Set(rc);
+  // TODO(addaleax): Throw `error` here. This isn't so terribly important,
```

Comment:
Hm, right, I could just do that here. I didn’t want to accidentally turn this into a semver-major change, but I guess here it really doesn’t make a difference… Updated!